crates/catalog/hms/Cargo.toml

```diff
@@ -33,5 +33,12 @@ anyhow = { workspace = true }
 async-trait = { workspace = true }
 hive_metastore = { workspace = true }
 iceberg = { workspace = true }
+log = "0.4.20"
```

Comment:
It's better to add in workspace instead.

Reply:
thank you - I changed it.